apps/backend/.env

```
@@ -118,6 +118,9 @@ HEXCLAVE_STRIPE_WEBHOOK_SECRET=# enter your stripe webhook secret
 HEXCLAVE_TELEGRAM_BOT_TOKEN= # enter you telegram bot token
 HEXCLAVE_TELEGRAM_CHAT_ID=# enter your telegram chat id
 
+# Voker
+VOKER_API_KEY=# enter your Voker API key from https://app.voker.ai
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)